cmd/start.go

```diff
@@ -275,3 +277,16 @@ func updateClient(ctx context.Context, bheUrl url.URL, bheClient *http.Client) e
 		}
 	}
 }
+
+func connectAndCreateClient() client.AzureClient {
```

Comment:
Tasty refactor. Can we move this to `utils.go`?

Reply:
Yeah, that's easy